CC_DV_R_bm3024_F.Rmd

```diff
@@ -0,0 +1,200 @@
+---
+title: "Creating interactive plot by Plotly"
+author: "Brian Mao"
+date: "3/8/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
```

Comment:
It would be best to move lines 8-11 down. As per 2.2.3 (https://jtr13.github.io/cc21/github-submission-instructions.html#submission-steps) the first line must contain a # followed by the title.